OpenWeatherMap/Model/Coordinate.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Coordinate.swift
+//  OpenWeatherMap
+//
+//  Created by Alevtina on 03.05.2020.
+//  Copyright © 2020 Alevtina. All rights reserved.
+//
+
+struct Coordinate: Codable, CustomStringConvertible {
+
+    var description: String {
+        return "Coordinates: \(longitude) lon, \(latitude) lat\n"
+    }
+
+    let longitude: Float
+    let latitude: Float
```

Comment:
let's use `Double` instead
```suggestion
    let longitude: Double
    let latitude: Double
```

Reply:
that's a good idea, thank you!